content/application-faq/application-stories.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Application stories from Gold Card Holders"
+weight: 6
+---
+
+To complement our FAQ, here some great online resources of current Gold Card Holders that explain how they obtained the GOld Card: 
```

Comment:
s/GO/Go

Reply:
Minor feedback:
To complement our FAQ, here **are** some great online resources of current Gold Card Holders that explain how they obtained the G**O**ld Card:

Reply:
```suggestion
To complement our FAQ, here are some great online resources of current Gold Card Holders that explain how they obtained the Gold Card: 
```